ouroboros-consensus-test/ouroboros-consensus-test.cabal

```diff
@@ -326,3 +326,45 @@ test-suite test-infra
                        -Wredundant-constraints
                        -Wmissing-export-lists
                        -fno-ignore-asserts
+
+benchmark bench-mempool
+  type:                exitcode-stdio-1.0
+  hs-source-dirs:      bench-mempool
+  main-is:             Main.hs
+  other-modules:       Bench.Consensus.Mempool
+                     , Bench.Consensus.MempoolWithMockedLedgerItf
+                     , Bench.Consensus.Mempool.TestBlock
+  build-depends:       base
+                     , cardano-slotting
+                     , containers
+                     , contra-tracer
+                     , deepseq
+                     , nothunks
+                     , QuickCheck
+                     , serialise         >=0.2   && <0.3
+                     , strict-stm
+                     , transformers
+                     , tree-diff
+                     , tasty-bench
+                     , tasty-hunit
+
+                     , ouroboros-consensus
+                     , ouroboros-consensus-test
+  default-language:    Haskell2010
+  ghc-options:         -Wall
+                       -Wcompat
+                       -Wincomplete-uni-patterns
+                       -Wincomplete-record-updates
+                       -Wpartial-fields
+                       -Widentities
+                       -Wredundant-constraints
+                       -Wmissing-export-lists
+                       -Wno-unticked-promoted-constructors
+                       -fno-ignore-asserts
```

Comment:
If the code under benchmark contains relatively costly assertions, not ignoring assertions might inflate the results. This is just something to take into consideration, it is not something that should change now

Reply:
True! I missed that. Thanks!